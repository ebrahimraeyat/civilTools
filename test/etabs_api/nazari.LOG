

 ETABS Ultimate 64-bit 19.1.0 Build 2420 (Analysis Build 9853/64)
 File: Y:\python\civiltools\civilTools\test\etabs_api\nazari.LOG




 B E G I N   A N A L Y S I S                                2021/07/25  19:38:08


 RUNNING ANALYSIS AS A SEPARATE PROCESS
 USING THE ADVANCED SOLVER (PROVIDES LIMITED INSTABILITY INFORMATION)

 NUMBER OF JOINTS                          =          67
    WITH RESTRAINTS                        =          15
    WITH MASS                              =          22
 NUMBER OF FRAME/TENDON ELEMENTS           =         122
 NUMBER OF SHELL ELEMENTS                  =          17
 NUMBER OF CONSTRAINTS/WELDS               =           4
 NUMBER OF LOAD PATTERNS                   =          22
 NUMBER OF ACCELERATION LOADS              =           9
 NUMBER OF LOAD CASES                      =          22

 ADDRESSABLE PHYSICAL MEMORY (RAM)         =       2.000 GB

 PARALLELIZATION OF ANALYSIS OPERATIONS:
  (Env. variable SAPFIRE_NUM_THREADS not set)
 NUMBER OF THREADS: STATE (AUTOMATIC)      =           1
 NUMBER OF THREADS: STIFFNESS (AUTOMATIC)  =           1
 NUMBER OF THREADS: EVENT (AUTOMATIC)      =           1
 NUMBER OF THREADS: MOVE (AUTOMATIC)       =           1
 NUMBER OF THREADS: RESPONSE (AUTOMATIC)   =           1
 NUMBER OF THREADS: SOLVE (AUTOMATIC)      =           1
 NUMBER OF THREADS: FORM (AUTOMATIC)       =           1


 N O N L I N E A R   S T A T I C   A N A L Y S I S                      19:38:08

 CASE: ~P-DELTA

 STARTING FROM ZERO (UNSTRESSED) INITIAL CONDITIONS

 LOAD CONTROL TYPE                         =       FORCE
 TYPE OF GEOMETRIC NONLINEARITY            =     P-DELTA
 USE EVENT STEPPING                        =         YES
 USE ITERATION                             =         YES
 USE LINE SEARCH                           =          NO
 EVENT LUMPING TOLERANCE                   =    0.010000
 FORCE CONVERGENCE TOLERANCE (RELATIVE)    =    0.000100


 E L E M E N T   F O R M A T I O N                                      19:38:08


 NUMBER OF COUPLED CONSTRAINT EQUATIONS    =           0

 Negative iterations are Constant-Stiffness
 Positive iterations are Newton-Raphson

  Saved    Null   Total  Iteration   Relative  Curr Step   Curr Sum    Max Sum
  Steps   Steps   Steps  this Step  Unbalance       Size   of Steps   of Steps
 (    1      50     200     -10/40   1.000000   1.000000   1.000000   1.000000)

      0       0       1         -1  35.825395   1.000000    .000000    .000000
      0       0       1  Conv   -2   0.605275   1.000000   1.000000   1.000000

 ----------------------------------------
 BASIC STABILITY CHECK FOR FORCE-CONTROLLED NONLINEAR STATIC LOAD CASES:
    NUMBER OF NEGATIVE STIFFNESS EIGENVALUES SHOULD BE ZERO FOR STABILITY.
    (NOTE: FURTHER CHECKS SHOULD BE CONSIDERED AS DEEMED NECESSARY)

    NUMBER FOUND AT FINAL CONVERGED STATE  =           0,  OK.
 ----------------------------------------

 NUMBER OF STIFFNESS FORMATIONS/SOLUTIONS  =           2

 TIME FOR INITIALIZING ANALYSIS            =        0.07
 TIME FOR CONTROLLING ANALYSIS             =        0.03
 TIME FOR UPDATING LOADS AND STATE         =        0.01
 TIME FOR FORMING STIFFNESS MATRIX         =        0.00
 TIME FOR SOLVING STIFFNESS MATRIX         =        0.02
 TIME FOR CALCULATING DISPLACEMENTS        =        0.02
 TIME FOR DETERMINING EVENTS               =        0.00
 TIME FOR SAVING RESULTS                   =        0.00
                                               ----------
 TOTAL TIME FOR THIS ANALYSIS              =        0.15


 E L E M E N T   F O R M A T I O N                                      19:38:08


 NUMBER OF COUPLED CONSTRAINT EQUATIONS    =           0


 L I N E A R   E Q U A T I O N   S O L U T I O N                        19:38:08

 FORMING STIFFNESS AT THE END OF CASE: ~P-DELTA

 TOTAL NUMBER OF EQUILIBRIUM EQUATIONS     =         189
 NUMBER OF NON-ZERO STIFFNESS TERMS        =        2808

 ---------------------------------
 BASIC STABILITY CHECK FOR LINEAR LOAD CASES:
    NUMBER OF NEGATIVE STIFFNESS EIGENVALUES SHOULD BE ZERO FOR STABILITY.
    (NOTE: FURTHER CHECKS SHOULD BE CONSIDERED AS DEEMED NECESSARY,
     SUCH AS REVIEWING EIGEN MODES FOR MECHANISMS AND RIGID-BODY MOTION)

    NUMBER OF NEGATIVE EIGENVALUES         =           0,  OK.
 ---------------------------------


 L I N E A R   S T A T I C   C A S E S                                  19:38:08

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 TOTAL NUMBER OF CASES TO SOLVE            =          14
 NUMBER OF CASES TO SOLVE PER BLOCK        =          14

 LINEAR STATIC CASES TO BE SOLVED:

 CASE: DEAD
 CASE: LIVE
 CASE: LRED
 CASE: LRED0.5
 CASE: LPARTITION
 CASE: LROOF
 CASE: SNOW
 CASE: WALL
 CASE: EX
 CASE: EY
 CASE: EV
 CASE: EXDRIFT
 CASE: EYDRIFT
 CASE: ~LLRF


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EXALL

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EYALL

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EXALL+0.3EY

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EXALL-0.3EY

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EYALL+0.3EX

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 M U L T I - S T E P   L I N E A R   S T A T I C   A N A L Y S I S      19:38:09

 CASE: EYALL-0.3EX

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF LOAD STEPS                      =           3


 Solving load steps:       1 to       3
 NUMBER OF OUTPUT STEPS SAVED              =           3


 E I G E N   M O D A L   A N A L Y S I S                                19:38:09

 CASE: MODAL

 USING STIFFNESS AT THE END OF CASE: ~P-DELTA

 NUMBER OF STIFFNESS DEGREES OF FREEDOM    =         189
 NUMBER OF MASS DEGREES OF FREEDOM         =          26
 MAXIMUM NUMBER OF EIGEN MODES SOUGHT      =          12
 MINIMUM NUMBER OF EIGEN MODES SOUGHT      =           1
 NUMBER OF RESIDUAL-MASS MODES SOUGHT      =           0
 NUMBER OF SUBSPACE VECTORS USED           =          24
 RELATIVE CONVERGENCE TOLERANCE            =    1.00E-09

 FREQUENCY SHIFT  (CENTER) (CYC/TIME)      =     .000000
 FREQUENCY CUTOFF (RADIUS) (CYC/TIME)      =  -INFINITY-
 ALLOW AUTOMATIC FREQUENCY SHIFTING        =         YES


 Original stiffness at shift : EV= 0.0000000E+00, f=     .000000, T=  -INFINITY-
 Number of eigenvalues below shift =      0
 Found mode      1 of     12:  EV= 1.9231024E+01, f=    0.697945, T=    1.432777
 Found mode      2 of     12:  EV= 2.7464717E+01, f=    0.834080, T=    1.198926
 Found mode      3 of     12:  EV= 4.5525081E+01, f=    1.073855, T=    0.931225
 Found mode      4 of     12:  EV= 1.7331265E+02, f=    2.095247, T=    0.477271
 Found mode      5 of     12:  EV= 2.0776526E+02, f=    2.294070, T=    0.435907
 Found mode      6 of     12:  EV= 2.6650231E+02, f=    2.598188, T=    0.384884
 Found mode      7 of     12:  EV= 4.3462283E+02, f=    3.318000, T=    0.301386
 Found mode      8 of     12:  EV= 5.5932134E+02, f=    3.764010, T=    0.265674
 Found mode      9 of     12:  EV= 6.2216508E+02, f=    3.969839, T=    0.251899
 Found mode     10 of     12:  EV= 1.3751012E+03, f=    5.901840, T=    0.169439
 Found mode     11 of     12:  EV= 1.7476975E+03, f=    6.653548, T=    0.150296
 Found mode     12 of     12:  EV= 1.8588774E+03, f=    6.861918, T=    0.145732

 NUMBER OF EIGEN MODES FOUND               =          12
 NUMBER OF ITERATIONS PERFORMED            =           4
 NUMBER OF STIFFNESS SHIFTS                =           0


 A N A L Y S I S   C O M P L E T E                          2021/07/25  19:38:09
