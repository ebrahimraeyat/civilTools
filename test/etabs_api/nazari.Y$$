  RespType: Case     ,  File: .Y00   ,  Records:                     0
                  3872
  RespType: Displ    ,  File: .Y01   ,  Records:                     0
                144720
  RespType: React    ,  File: .Y03   ,  Records:                     0
                144720
  RespType: Frame    ,  File: .Y05   ,  Records:                     0
                 37626
  RespType: Shell    ,  File: .Y06   ,  Records:                     0
                 16609
  RespType: Spring   ,  File: .Y02   ,  Records:                     0
                   804
  RespType: JFJoint  ,  File: .Y09   ,  Records:                     0
                141504
  RespType: JFSpring ,  File: .Y0A   ,  Records:                     0
                141504
  RespType: StrainAge,  File: .Y04   ,  Records:                     0
                  2224
  RespType: DisplTran,  File: .Y0B   ,  Records:                     0
                 38592
  StateType: Frame    ,  File: .Y07   ,  Records:                     0
                 56120
  StateType: Shell    ,  File: .Y08   ,  Records:                     0
                  4148
