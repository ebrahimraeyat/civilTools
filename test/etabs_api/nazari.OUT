

 ETABS Ultimate 64-bit 19.1.0 Build 2420 (Analysis Build 9853/64)               
 File: Y:\python\civiltools\civilTools\test\etabs_api\nazari.OUT                
                                                                     
                                                                       

 1, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
           19.049401   19.049401     .000000     .000000     .000000  2.1064E+08

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     11624.487   11624.487   11104.081
     Y      6334.732    6334.732    6222.455
     Z     15400.000   15400.000   15400.000

 2, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
           92.312388   92.312388     .000000     .000000     .000000  2.7867E+09

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     15980.598   15980.598   16620.046
     Y      3600.201    3600.201    4031.554
     Z     12160.000   12160.000   12160.000

 3, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
          106.053274  106.053274     .000000     .000000     .000000  3.2287E+09

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     15769.227   15769.227   16603.787
     Y      3413.658    3413.658    4017.204
     Z      8740.000    8740.000    8740.000

 4, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
          181.381270  181.381270     .000000     .000000     .000000  1.0893E+10

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     11603.432   11603.432   13454.320
     Y      3811.083    3811.083    4318.887
     Z      5320.000    5320.000    5320.000

 1, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
           19.049401   19.049401     .000000     .000000     .000000  2.1064E+08

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     11624.487   11624.487   11104.081
     Y      6334.732    6334.732    6222.455
     Z     15400.000   15400.000   15400.000

 2, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
           92.312388   92.312388     .000000     .000000     .000000  2.7867E+09

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     15980.598   15980.598   16620.046
     Y      3600.201    3600.201    4031.554
     Z     12160.000   12160.000   12160.000

 3, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
          106.053274  106.053274     .000000     .000000     .000000  3.2287E+09

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     15769.227   15769.227   16603.787
     Y      3413.658    3413.658    4017.204
     Z      8740.000    8740.000    8740.000

 4, CONSTR = RIGID, DOF = U1 U2 R3

                     LOCAL COORDINATE SYSTEM FOR CONSTRAINT MASTER
  GLOBAL          U1          U2          U3          R1          R2          R3
     X      1.000000     .000000     .000000    1.000000     .000000     .000000
     Y       .000000    1.000000     .000000     .000000    1.000000     .000000
     Z       .000000     .000000    1.000000     .000000     .000000    1.000000

                     TRANSLATIONAL MASS AND MASS MOMENTS OF INERTIA
                  U1          U2          U3          R1          R2          R3
          181.381270  181.381270     .000000     .000000     .000000  1.0893E+10

                     CENTER OF MASS
  GLOBAL          U1          U2          U3
     X     11603.432   11603.432   13454.320
     Y      3811.083    3811.083    4318.887
     Z      5320.000    5320.000    5320.000

 D I S P L A C E M E N T   D E G R E E S   O F   F R E E D O M

         (A) = Active DOF, equilibrium equation
         (-) = Restrained DOF, reaction computed
         (+) = Constrained DOF
         (>) = External substructure DOF
         ( ) = Null DOF

  JOINTS              UX  UY  UZ  RX  RY  RZ
       1               +   +   A   A   A   +
       5               -   -   -   -   -   -
       9               +   +   A   A   A   +
      10               -   -   -   -   -   -
      11               A   A   A   A   A   A
      12               +   +   A   A   A   +
      13               A   A   A   A   A   A
      14               +   +   A   A   A   +
      15               -   -   -   -   -   -
      16 TO      19    +   +   A   A   A   +
      20               -   -   -   -   -   -
      21 TO      24    +   +   A   A   A   +
      25               -   -   -   -   -   -
      29               +   +   A   A   A   +
      30               -   -   -   -   -   -
      31 TO      34    +   +   A   A   A   +
      35               -   -   -   -   -   -
      36 TO      38    +   +   A   A   A   +
      39               -   -   -   -   -   -
      40               +   +   A   A   A   +
      41               A   A   A   A   A   A
      42               +   +   A   A   A   +
      43               A   A   A   A   A   A
      44               -   -   -   -   -   -
      45 TO      48    +   +   A   A   A   +
      49               -   -   -   -   -   -
      50 TO      53    +   +   A   A   A   +
      54               -   -   -   -   -   -
      55 TO      80    +   +   A   A   A   +
      88               A   A   A   A   A   A
      89 TO      92    +   +   A   A   A   +
      93               A   A   A   A   A   A
      94               +   +   A   A   A   +
      95               A   A   A   A   A   A
       2 TO       6    +   +   -   -   -   +
    NULL                                    

  CONSTRAINTS         U1  U2  U3  R1  R2  R3
 1       TO 4          A   A               A
